### DISCUSSION TITLE
Sayı dizisinde soru işaretli yere ne gelmelidir sorusu

### DISCUSSION THREAD

**Discussion by sonsuzus** [sonsuzus/sonsuzus#237] [Zeka Sorusu]
Soru işareti yerine ne gelmelidir?
ÜÇ, BİR, DÖRT, SEKİZ, ON DÖRT, ON SEKİZ, YİRMİ BİR, ?

**Comment by Berkek007** [comment]
CEVAP: YİRMİ DÖRT, çünkü üç (iki harften), bir (üç harften), dört(dört harften) , sekiz(beş harften) olacak şekilde birer artarak gittiği için yirmi bir(8 harf) sayısını 9 harfli olan yirmi dört sayısı takip etmelidir.
